### PR TITLE
Chess set

### PR DESCRIPTION
**Pull request by PRTBRM**

### CODE REVIEW

**Comment by PRTBRM** [comment]
Hey so I'm not sure what Vercel is, and this is my first pull request. Did I not do something correctly?

File: libraries.json [hunk 1]
```diff
@@ -679,4 +679,17 @@
     "preview": "pgilfernandez/basic-shapes.png",
     "date": "2021-06-22"
   }
```
Comment (ad1992) [inline libraries.json:681]:
```suggestion
  },
```

**Comment by ad1992** [comment]
> Hey so I'm not sure what Vercel is, and this is my first pull request. Did I not do something correctly?

It's fixed now, there was a syntax error in `libraries.json` due to which the deployment was failing

**Comment by dwelle** [comment]
Got bored so I created an alternative chess pieces. Take it or leave it.

https://excalidraw.com/#json=6214175047024640,yvM0RBAH1AxWJDtibzOl_g

image

**Comment by PRTBRM** [comment]
> Got bored so I created alternative chess pieces. Take it or leave it.
> 
> https://excalidraw.com/#json=6214175047024640,yvM0RBAH1AxWJDtibzOl_g
> 
> image

Yes @dwelle!! That's what I was trying to create but couldn't figure out how to use my Apple Pencil very effectively. So I went with just using shapes. Ok cool - so what are the next steps to use your pieces instead? I am super new to git, but I want to see this in Excalidraw!! thank you

**Comment by dwelle** [comment]
> So I went with just using shapes. Ok cool - so what are the next steps to use your pieces instead? I am super new to git, but I want to see this in Excalidraw!! thank you

No prob, I can add them to the library later, and fix the conflict as well.

**Comment by dwelle** [comment]
@PRTBRM I've also updated the chess board itself. Check it out. If everything's looking good, we can merge.

**Review by ad1992 — APPROVED** [review]
Woohoo this looks amazing 🔥

**Comment by PRTBRM** [comment]
Fantastic!! So is there anything I need to do for this to get merged and then be available in the library?

**Comment by dwelle** [comment]
Merged :)

**Pull request merged**